wrappers/python/openmm/app/simulation.py

```diff
@@ -142,8 +142,11 @@ def minimizeEnergy(self, tolerance=10*unit.kilojoules_per_mole/unit.nanometer, m
         """
         mm.LocalEnergyMinimizer.minimize(self.context, tolerance, maxIterations, reporter)
 
-    def step(self, steps):
+    def step(self, steps: int):
         """Advance the simulation by integrating a specified number of time steps."""
+        if not isinstance(steps, int):
```

Comment:
How about changing the check to `if int(steps) != steps`?  That way it will accept any integer type, for example `np.int64`, not just `int`.

Reply:
Good point. I also like this approach as it will not break old simulation setups where a float f%1=0 was passed.